server/controllers/comment.controller.js

```diff
@@ -22,9 +27,16 @@ class CommentController {
     const { id } = req.user;
     const { slug } = req.params;
     const { comment } = req.body;
-    const newcomment = await comments.create({
-      userid: id,
-      articleid: slug,
+
+    const findArticle = await Article.findOne({
+      where: { slug },
+    });
+    if (!findArticle) return Response(res, 404, 'Article not found.');
+
+    const table = findArticle.dataValues.published ? comments : DraftComment;
```

Comment:
Why is there no consistency in the table naming? 
We have `comments` and `DraftComment` tables.

Reply:
@paradisekelechi David worked on creating the comments table and I did not want to edit his code. Should i go ahead and fix it?

Reply:
It's required that there be consistency. Check what the standard is and effect the required changes. @TheDrizzyWay